docs/development_guide.md

```diff
@@ -185,10 +146,7 @@ The different chart templates are used as follows:
 - `prometheus-*.yaml` chart templates are used to deploy a Prometheus instance when the metrics stack is enabled
 - `zipkin-*.yaml` chart templates are used to deploy a Zipkin instance when Zipkin tracing is enabled
 
+## Custom Resource Definitions
```

Comment:
Should be a subsection of `Helm charts`

Reply:
I added another `#` to indent under `Helm charts`
Thank you!